Netable/Example/Services/NetworkService.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  SimpleNetworkService.swift
+//  Example
+//
+//  Created by Amy Oulton on 2022-11-28.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import Combine
+import Foundation
+import Netable
+
+class SimpleNetworkService {
+    static var shared = SimpleNetworkService()
+
+    private let netable = Netable(baseURL: URL(string: "http://localhost:8080/")!)
+
+    func getVersion() {
```

Comment:
We might consider changing this to be async and bubble the errors up a level to be consistent with our other services

Reply:
I wrote this one using the Task do demonstrate try/catch within the actual service because that was listed in the project requirements! If we want to be consistent I can definitely change that. We are using try catch within the `createPost` request in the `authNetworkService`!

Reply:
okay, lets just leave it. I hadn't intended to specify where users were supposed to use try/catch to handle errors, and our idiomatic solution is usually to pass errors back up to the VM but might as well show both ways